docs/support-guide.md

```diff
@@ -0,0 +1,243 @@
+
+- `failures-baseline/${PLUGIN_NAME}_${INDEX}-failure.txt`: the error summary
+- `failures-baseline/${PLUGIN_NAME}_${INDEX}-systemOut.txt`: the entire stdout of the failed plugin
+
+Considerations:
+
+- `${PLUGIN_NAME}`: currently these plugins names are valid: [`openshift-validated`, `kubernetes-conformance`]
+- `${INDEX}` is the simple index ordered by test name on the list
+
+> TODO: mention the spreadsheet `failures-index.xlsx` created. It can be used to take notes of what was already reviewed by each layer/stakeholder (partner, support, and engineering) reviewing the errors, and also to improve the communication between the parts. For example, the partner could provide that google sheet with notes to check what he already reviewed (as we [mrbraga and robert] do while reviewing, and submitting it). The support can keep that list updated sharing it with engineering, and so on.
+
+Example of files on the extracted directory:
+
+> Review if we need it
+
+```bash
+$ tree processed/
+processed/
+├── failures-baseline
+[redacted]
+├── failures-provider
+[redacted]
+├── failures-provider-filtered
+│   ├── kubernetes-conformance_1-1-failure.txt
+│   ├── kubernetes-conformance_1-1-systemOut.txt
+│   ├── kubernetes-conformance_2-2-failure.txt
+│   ├── kubernetes-conformance_2-2-systemOut.txt
+│   ├── openshift-validated_1-31-failure.txt
+│   ├── openshift-validated_1-31-systemOut.txt
+[redacted]
+│   ├── openshift-validated_7-1-failure.txt
+│   └── openshift-validated_7-1-systemOut.txt
+├── tests_kubernetes-conformance_baseline_failures.txt
+├── tests_kubernetes-conformance_provider_failures.txt
+├── tests_kubernetes-conformance_provider_filter1-suite.txt
+├── tests_kubernetes-conformance_provider_filter2-baseline.txt
+├── tests_kubernetes-conformance_suite_full.txt
+├── tests_openshift-validated_baseline_failures.txt
+├── tests_openshift-validated_provider_failures.txt
+├── tests_openshift-validated_provider_filter1-suite.txt
+├── tests_openshift-validated_provider_filter2-baseline.txt
+└── tests_openshift-validated_suite_full.txt
+
+3 directories, 300 files
+```
+
+### Review Guidelines <a name="release"></a>
+
+This section is a guide of the initial files to review when start exploring the resulting archive.
+
+Items to review:
+
+- OCP version matches the certification request
+- Review the result file
+- Check if the failures are 0, if not, need to check one by one
+- To provide a better interaction between the review process, one spreadsheet named `failures-index.xlsx` is created inside the extracted directory (`./processed/` exemplified in the last section). It can be used as a tool to review failures and take notes about them.
+- Check details of each test failed on the sub-directory `failures-provider-filtered/*.txt`.
+
+## <TODO>
+
+> I am thinking to move the troubleshooting section from user Doc to here or to "user-result-review.md" to avoid keeping useful commands potentially used by Partners in the "Support" Document. Ideas?
```

Comment:
Agreed,

Reply:
@bostrt I think that moving to a troubleshooting document could be a good idea. That kind of document can grow so fast, keeping a dedicated document for that purpose could be a good idea. We also will keep the "Main User Guide" clean and dedicated to the functional steps of OPCT.